tests/test_animation.py

```diff
@@ -102,14 +107,17 @@ def test_send_animation_url_file(self, bot, chat_id, animation):
 
         assert isinstance(message.animation, Animation)
         assert isinstance(message.animation.file_id, str)
-        assert message.animation.file_id is not None
+        assert isinstance(message.animation.file_unique_id, str)
+        assert message.animation.file_id != ''
+        assert message.animation.file_unique_id != ''
+
         assert message.animation.duration == animation.duration
+        assert message.animation.file_name == animation.file_name
         assert message.animation.mime_type == animation.mime_type
         assert message.animation.file_size == animation.file_size
 
     @flaky(3, 1)
     @pytest.mark.timeout(10)
-    @pytest.mark.skip(reason='Doesnt work without API 4.5')
```

Comment:
can you enlighten me why this is in in the first place? As far as I understood. This should just work without 4.5...

Reply:
Since API 4.5 any file may have multiple `file_id` s and the only valid comparison is `file_unique_id`. The latter is not present in master and hence tests will fail.